graphql_compiler/schema_generation/graphql_schema.py

```diff
@@ -15,6 +15,21 @@
 from .exceptions import EmptySchemaError
 
 
+def _has_only_vertex_non_abstract_subclasses(cls_name, schema_graph):
+    """Return True if all the classes non-abstract subclasses are vertices. Else, return False."""
+    cls_subclasses = schema_graph.get_subclass_set(cls_name)
+
+    all_non_abstract_subclasses_are_vertices = True
+    for subclass_name in cls_subclasses:
+        subclass = schema_graph.get_element_by_class_name(subclass_name)
```

Comment:
`subclass` might get confused for being a class object, rather than a `SchemaElement`. Consider `subclass_element` as an alternative name?